src/main/scala/com/johnsnowlabs/reader/TextReader.scala

```diff
@@ -23,6 +23,57 @@ import org.apache.spark.sql.functions.udf
+  *   Threshold ratio of empty lines used to decide between new line-based or broken-paragraph
+  *   grouping. Default value is 0.1.
+  *
+  * ==Example==
+  * {{{
+  * val filePath = "home/user/txt/files"
+  * val textReader = new TextReader()
+  * val textDf = textReader.txt(filePath)
+  * }}}
+  *
+  * {{{
+  * textDf.select("txt").show(false)
+  * +-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+  * |txt                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+  * +-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+  * |[{Title, BIG DATA ANALYTICS, {paragraph -> 0}}, {NarrativeText, Apache Spark is a fast and general-purpose cluster computing system.\nIt provides high-level APIs in Java, Scala, Python, and R., {paragraph -> 0}}, {Title, MACHINE LEARNING, {paragraph -> 1}}, {NarrativeText, Spark's MLlib provides scalable machine learning algorithms.\nIt includes tools for classification, regression, clustering, and more., {paragraph -> 1}}]|
+  * +-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+  *
+  * textDf.printSchema()
+  * root
+  *  |-- path: string (nullable = true)
+  *  |-- txt: array (nullable = true)
+  *  |    |-- element: struct (containsNull = true)
+  *  |    |    |-- elementType: string (nullable = true)
+  *  |    |    |-- content: string (nullable = true)
+  *  |    |    |-- metadata: map (nullable = true)
+  *  |    |    |    |-- key: string
+  *  |    |    |    |-- value: string (valueContainsNull = true)
+  * }}}
+  */
```

Comment:
It might be helpful to include a reference link to the TextReader demo notebook that has examples with all available parameters: `examples/python/reader/SparkNLP_Text_Reader_Demo.ipynb`